robottelo/ui/activationkey.py

```diff
@@ -25,11 +23,9 @@ def set_limit(self, limit):
             self.find_element(limit_checkbox_locator).click()
             self.field_update("ak.usage_limit", limit)
 
-    def create(self, name, env, limit=None, description=None,
+    def create(self, name, env=None, limit=None, description=None,
```

Comment:
I'm wondering if this change affects any of the existing tests now that ``env`` is optional?